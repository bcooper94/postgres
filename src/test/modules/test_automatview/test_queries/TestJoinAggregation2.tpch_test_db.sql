SET automatview.training_sample_count = 1;

select
	c_custkey,
	c_name,
	sum(l_extendedprice),
	c_acctbal,
	n_name,
	c_address,
	c_phone,
	c_comment
from
	customer join orders on c_custkey = o_custkey
	join lineitem on o_orderkey = l_orderkey
	join nation on c_nationkey = n_nationkey
group by
	c_custkey,
	c_name,
	c_acctbal,
	c_phone,
	n_name,
	c_address,
	c_comment;

select
	c_custkey,
	c_name,
	sum(l_extendedprice),
	c_acctbal,
	n_name,
	c_address,
	c_phone,
	c_comment
from
	customer join orders on c_custkey = o_custkey
	join lineitem on o_orderkey = l_orderkey
	join nation on c_nationkey = n_nationkey
group by
	c_custkey,
	c_name,
	c_acctbal,
	c_phone,
	n_name,
	c_address,
	c_comment;
