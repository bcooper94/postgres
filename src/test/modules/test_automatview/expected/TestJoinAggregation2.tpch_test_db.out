1
SELECT Matview.customer_c_custkey AS Matview_customer_c_custkey, Matview.customer_c_name AS Matview_customer_c_name, Matview.sum_lineitem_l_extendedprice AS Matview_sum_lineitem_l_extendedprice, Matview.customer_c_acctbal AS Matview_customer_c_acctbal, Matview.nation_n_name AS Matview_nation_n_name, Matview.customer_c_address AS Matview_customer_c_address, Matview.customer_c_phone AS Matview_customer_c_phone, Matview.customer_c_comment AS Matview_customer_c_comment FROM Matview